src/main.cpp

```diff
@@ -572,6 +572,20 @@ int main ( int argc, char** argv )
             continue;
         }
 
+        // Clean up legacy fader settings --------------------------------------
+        // Undocumented temporary command line argument: Clean up fader settings
```

Comment:
Since we agreed to remove this soon, I'd prefer to document it in the help text.

Reply:
It's going to be totally meaningless to most users.  To those for whom it's applicable, they'll use it once and never again.  I can't see the point of making the help text get even larger than it is now for this.

Reply:
> If it shows up under --help, the version supports it,

If running the app with the option doesn't complain it's not recognised, it's supported, too.

Also, if it goes into help, that means we should update the website User Guide, too.

Reply:
> If running the app with the option doesn't complain it's not recognised, that means it's supported, too.

Fair point. 

I'd still like to have it prominently documented somewhere besides the release announcement. I assume most users don't read the announcement at all.

Reply:
True, true.  We could add it to the hints and tips about `--ctrlmidich`?  It's less prominent but it's more relevant.

Reply:
Probably? 
What about printing it to stdout if the --ctrlmidich option is set and to the website as long as it's in?

Reply:
We could write a forum post with a how to?

Reply:
This all arose in https://github.com/orgs/jamulussoftware/discussions/2678 -- that could have a comment added with how to fix it.

Reply:
I've noted on the _Reminders for the release announcement_ to comment on the original discussion, so going ahead with the merge here.